gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/CredentialsController.groovy

```diff
@@ -97,11 +97,11 @@ class CredentialsController {
     @ApiParam(value = 'Value of the "@type" key for accounts to search for.', example = 'kubernetes')
     @PathVariable String accountType,
     @ApiParam('Maximum number of entries to return in results. Used for pagination.')
-    @RequestParam OptionalInt limit,
+    @RequestParam(required = false, defaultValue = "100") Integer limit,
```

Comment:
The previous code had no default, right?

Reply:
Ah that's right, I fixed it now.